install.sh

```diff
@@ -3525,6 +3525,17 @@ disable_tty_input() {
     exec </dev/null
 }
 
+############################################################
+# Checks whether the given directory
+#   is a Git repository (bare included) or not.
+#
+# Returns:
+#   1 if failed, 0 otherwise
+############################################################
+is_git_repo() {
+    (cd "$1" && git rev-parse --is-bare-repository >/dev/null 2>&1) || return 1
```

Comment:
I don't understand why this didn't break tests. :thinking: 
I just run `git rev-parse --is-bare-repository` on a local repository and it returns `false`, but it is a Git repo.

Reply:
Really, which git version, hm.. probably not such a good idea.

Reply:
Yeah, but the point is, you only want to check if its a git repo :)
This function works because if its not a git repo (bare or not)
`git rev-parse --is-bare-repository` returns not `1`

Reply:
This should be faster, then `git status` which might really slow down things. and works for both bare and non-bare.